ui/raidboss/data/04-sb/dungeon/fractal_continuum_hard.ts

```diff
@@ -216,20 +234,26 @@ export default {
     },
     {
       id: 'Fractal Hard Dischord Collect',
+      type: 'HeadMarker',
       netRegex: NetRegexes.headMarker({ id: ['004D', '004E'] }),
-      run: (data, matches) => data[matches.id] = matches.target,
```

Comment:
I could have kept this the same, storing `004D` and `004E` properties on `data` directly, but decided to give it its own field like everything else.